nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -398,10 +398,14 @@ def add_electrodes(
     for name in elec_columns_append:
         _ = elec_columns.pop(name)
 
+    # add channel_id column
+    nwbfile.add_electrode_column(name="channel_id", description="channel_id", index=False)
```

Comment:
I was playing with this and it gives problems in the case were the same `nwb` file is re-written by another extractor.
Maybe?
```suggestion
    if 'channel_id' not in nwbfile.electrodes.colnames:
        nwbfile.add_electrode_column(name="channel_id", description="channel_id", index=False)
```

Reply:
> I went through the discussion of this in the slack. Wasn't @CodyCBakerPhD suggestion there to create a new property called 'channel_name' that would be the string version of channel_id?

Indeed; the electrode table also has a primary ID for each electrode, the field `electrode.id`, a unique integer. Naming this new column `channel_id` could then get confusing if the two differ, whereas an aliasing column `channel_name` would be the arbitrary string version of the associated ID.

One question is whether to bother writing the column `channel_name` in the case where they correspond one to one (`int(electrode.channel_name) == electrode.id`). A point in favor of this would be to allow better representation of existing electrodes when appending new electrodes that do have differing names from the IDs.

Reply:
Also as per below, it would be nice for us to internally rely on the existence of the `channel_name` column so we can always form an index map between `channel_name` and `id`.